sqlalchemy_hana/dialect.py

```diff
@@ -555,6 +560,22 @@ def get_table_oid(self, connection, table_name, schema=None, **kw):
         table_oid = (result.fetchone())[0]
         return table_oid
 
+    def get_table_comment(self, connection, table_name, schema=None, **kw):
+        schema = schema or self.default_schema_name
+
+        result = connection.execute(
+            sql.text(
+                "SELECT COMMENTS FROM TABLES WHERE SCHEMA_NAME=:schema AND TABLE_NAME=:table"
+            ).bindparams(
+                schema=self.denormalize_name(schema),
+                table=self.denormalize_name(table_name),
+            )
+        )
+        text = {}
```

Comment:
Based on the other implementations it looks like `get_table_comment` should always return a dict with key `text`. In case the table doesn't exist oder the comment is NULL then the value should be `None`.